frontend/nuxt.config.js

```diff
@@ -13,6 +13,7 @@ export default {
         PRODUCTION_DOMAIN: process.env.PRODUCTION_DOMAIN,
         CONTACT_EMAIL: process.env.CONTACT_EMAIL || undefined,
         GITHUB_URL: pkg.repository.url,
+        IS_STAGING_ENVIRONMENT: process.env.IS_STAGING_ENVIRONMENT || false,
```

Comment:
You also probably want to convert this variable, since normal env variables are always comes as strings:
```suggestion
        IS_STAGING_ENVIRONMENT: Boolean(process.env.IS_STAGING_ENVIRONMENT) || false,
```